source/extensions/omni.isaac.lab/omni/isaac/lab/sim/simulation_context.py

```diff
@@ -383,15 +386,14 @@ def reset(self, soft: bool = False):
             for _ in range(2):
                 self.render()
 
-    def step(self, render: bool = True):
-        """Steps the physics simulation with the pre-defined time-step.
+    def step(self):
```

Comment:
This is a huge breaking change. We should be care with all the other logics/workflows.